packages/terra-collapsible-menu-view/package.json

```diff
@@ -40,7 +41,7 @@
     "terra-icon": "^3.1.0"
   },
   "scripts": {
-    "compile": "babel --root-mode upward --out-dir lib --copy-files",
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
```

Comment:
Is this necessary?

Reply:
Yes it is required. Even though a repo level "compile" would run successfully, if we try to "compile" a single package without specifying "src" the following error is thrown `--out-dir requires filenames`.

Reply:
Does this need to be changed for all packages?

Reply:
It sounds that way, let's get an issue logged for this @avinashg1994 and then we can get it resolved for the other packages in a separate PR.

Reply:
I scanned the entire repo and found that only **Collapsible Menu View** and **Tabs** required this change, which has been addressed as part of this PR.
All the other packages do not require this change.